<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWidget</class>
 <widget class="QWidget" name="GraphWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>676</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Volume TF</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="controlsWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Active channel:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbAlpha">
          <property name="text">
           <string>a&amp;lpha</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbRed">
          <property name="text">
           <string>red</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbGreen">
          <property name="text">
           <string>&amp;green</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbBlue">
          <property name="text">
           <string>bl&amp;ue</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>463</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rbAlpha</sender>
   <signal>toggled(bool)</signal>
   <receiver>GraphWidget</receiver>
   <slot>OnChannelButtonToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbRed</sender>
   <signal>toggled(bool)</signal>
   <receiver>GraphWidget</receiver>
   <slot>OnChannelButtonToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbGreen</sender>
   <signal>toggled(bool)</signal>
   <receiver>GraphWidget</receiver>
   <slot>OnChannelButtonToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbBlue</sender>
   <signal>toggled(bool)</signal>
   <receiver>GraphWidget</receiver>
   <slot>OnChannelButtonToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnChannelButtonToggled(bool)</slot>
 </slots>
</ui>
